examples/tts/conf/talknet-pitch.yaml

```diff
@@ -0,0 +1,159 @@
+name: &name TalkNetPitch
+sample_rate: &sr 22050
```

Comment:
Please remove all & and * yaml anchors and use the hydra anchors instead. See https://github.com/NVIDIA/NeMo/blob/main/examples/tts/conf/tacotron2.yaml for an example